src/Microsoft.R.Host.vcxproj

```diff
@@ -102,6 +102,7 @@
       <SDLCheck>false</SDLCheck>
       <AdditionalIncludeDirectories>..\lib\picojson;..\lib\websocketpp</AdditionalIncludeDirectories>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
+      <DisableSpecificWarnings>4200</DisableSpecificWarnings>
```

Comment:
This is for 0-length array in `message_repr`. It's technically non-standard C++, but VC++, gcc and Clang all support it these days, and it makes for more readable code than otherwise.